controllers/consumer_controller.go

```diff
@@ -75,27 +75,36 @@ type ConsumerReconciler struct {
 // +kubebuilder:rbac:groups=apps,resources=deployments/status,verbs=get
 
 func (r *ConsumerReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	start := time.Now()
+	defer func() {
+		reconcileDuration.WithLabelValues(req.Namespace).Observe(time.Since(start).Seconds())
```

Comment:
I think we should have `consumer.Name` in every metrics instead of the namespace.